sdk/eventhub/event-hubs/samples/browserSample/README.md

```diff
@@ -64,6 +64,12 @@ npm start
 
 4. Navigate to the web page by visiting http://localhost:8080 in a browser.
 
+## Troubleshooting
+
+### Authentication error: AADSTS50011
+
+If you receive error `AADSTS50011` with the message `The reply URL specified in the request does not match the reply URLs configured for the application`, make sure that you're accessing the sample using `localhost` and not an IP address (e.g. 127.0.0.1).
```

Comment:
Make sure that you're accessing the sample using the same address that is configured as the reply URL for the application?

Reply:
Added some more wording around this, and an explicit prerequisite to set the redirect uri. (Yes, I know in the app registration you set the redirect uri, and in the error message it says reply uri, but this is how you resolve it)